### PR TITLE
Add release notes for v1.12.0

### PR DESCRIPTION
**Pull request by denis-tingaikin**

### CODE REVIEW

File: content/docs/releases/v1.11.0.md [hunk 1]
```diff
@@ -19,26 +19,52 @@ NSM v1.11.0 has been tested on:
...
+[See more details](https://github.com/networkservicemesh/integration-tests/issues/2862)
+
+
 ### Add LoadBalancer for vl3 networks
 
-NSM vl3 could be perceived as an alternative to k8s networks. Currently, k8s service is the most useful thing in the k8s network.
+NSM vl3 could be perceived as alternative of k8s networks. Currently, k8s service is most useful thing in k8s network. 
 
-At this moment, NSM vl3 network doesn't provide any alternative to k8s services. So we need to consider and implement the best solution for it.
+At this moment, NSM vl3 netowk doesn't provide any alternative of k8s services. So we need to consider and implement the best solution for it.
```
Comment (NikitaSkrynnik) [inline content/docs/releases/v1.11.0.md:35]:
```suggestion
At this moment, NSM vl3 network doesn't provide any alternative of k8s services. So we need to consider and implement the best solution for it.
```

File: content/docs/releases/v1.12.0.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+- [AKS](https://github.com/networkservicemesh/integration-k8s-aks/actions?query=branch%3Arelease%2Fv1.12.0+)
+- [AWS/AWS IPv6](https://github.com/networkservicemesh/integration-k8s-aws/actions?query=branch%3Arelease%2Fv1.12.0+)
+- [Bare metal in Equinix Metal](https://github.com/networkservicemesh/integration-k8s-packet/actions?query=branch%3Arelease%2Fv1.12.0+)
+- [Interdomain GKE/AWS/AKS](https://github.com/networkservicemesh/integration-interdomain-k8s/actions?query=branch%3Arelease%2Fv1.12.0+)
+
+## Changes since last release
+
+
+
+### `begin` can't do `Unregister` requests without `Register` requests
```
Comment (NikitaSkrynnik) [inline content/docs/releases/v1.12.0.md:24]: It's been made in v1.11.0. I think we should delete it from v1.12.0

File: content/docs/releases/v1.12.0.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+
+Status: RESOLVED.
+
+[See more details](https://github.com/networkservicemesh/sdk/issues/1515)
+
+
+### Registry k8s can not correctly handle unregister when its scaled
```
Comment (NikitaSkrynnik) [inline content/docs/releases/v1.12.0.md:31]: This one has been done in v1.11.0 too. https://github.com/networkservicemesh/site/pull/276/files#diff-2e7b5ea6e25a866a76bdcb081f79ee9b01cfe8280bc03ccfefed16e048a682ffR99

File: content/docs/releases/v1.12.0.md (hunk 2 above)
Comment (denis-tingaikin) [reply content/docs/releases/v1.12.0.md]: Fixed

File: content/docs/releases/v1.11.0.md (hunk 1 above)
Comment (denis-tingaikin) [reply content/docs/releases/v1.11.0.md]: Thanks, fixed.

**Review by glazychev-art — APPROVED** [review]

**Pull request merged**